index.js

```diff
@@ -101,7 +102,8 @@ async function authorize(options) {
   const postOptions = {
     method: 'POST',
     body: form,
-    headers: form.getHeaders()
+    headers: form.getHeaders(),
+    agent
```

Comment:
technically node-fetch custom agent can take in a function as well (see node-fetch docs)
Documentation is missing in README.md regarding the agent option.